src/cleaning/gather.jl

```diff
@@ -0,0 +1,60 @@
+# ------------------------------------------------------------------
+# Licensed under the MIT License. See LICENSE in the project root.
+# ------------------------------------------------------------------
+
+"""
+    gather(mesh)
+
+Merge the duplicated vertices of `mesh`.
```

Comment:
I don't understand what the function does. Also, we can't merge PRs with multiple additions. We need a PR for a single function with well-defined scope and good tests.

Reply:
See the example in the unit tests, you will understand.